mongomock/collection.py

```diff
@@ -74,6 +76,14 @@ class ReturnDocument(object):
 }
 
 
+def _bson_encode(document, codec_options):
+    if isinstance(codec_options, CodecOptions):
+        BSON.encode(document, check_keys=True,
```

Comment:
Please break after `encode(`

Reply:
Time to add ruff format lint stage to the CI? :)